Feature/Education.feature

```diff
@@ -0,0 +1,20 @@
+﻿Feature: Education
+User wants to add education details into Profile Details
+Acceptance Criteria
+User is able to add the education details
+
+@mytag
+ Scenario: 01 Test Add Education sucessfully
+	Given User enter UniversityName as "Delhi University"
+	And   Select CountryOfUniversity as "India"
+	And   Select Title as "B.Tech"
+	And   Enter Degree as "Computer Science"

```

Comment:
Please use the scenario outline to add the data. This is the standard. Please check available examples online.